sensitivity_games/controller.py

```diff
@@ -13,6 +13,16 @@ def get_control(self, x):
         raise NotImplementedError('Not Implemented!')
 
 
+class RegulatorController(Controller):
```

Comment:
nit: this and next class should be in separate files.... but not a huge deal of course

Reply:
ah, right, i'll fix.